blocks/identity-block/features/account-management/default.jsx

```diff
@@ -81,26 +88,52 @@ function AccountManagement({ customFields }) {
   }
 
   const header = phrases.t('identity-block.account-information');
+  const socialProfileHeader = phrases.t('identity-block.connected-accounts');
 
   // if logged in, return account info
   return (
-    <AccountManagementPresentational header={header}>
+    <>
+      <AccountManagementPresentational header={header}>
+        {
+          showEmail && (
+            <EmailEditableFieldContainer
+              email={email}
+              setEmail={setEmail}
+            />
+          )
+        }
+        {
+          showPassword ? (
+            <PasswordEditableFieldContainer
+              email={email}
+              hasPassword={hasPassword}
+              setHasPassword={setHasPassword}
+            />
+          ) : null
+        }
+      </AccountManagementPresentational>
       {
-        showEmail && (
-          <EmailEditableFieldContainer
-            email={email}
-            setEmail={setEmail}
-          />
-        )
+        showSocialProfile ? (
+          <AccountManagementPresentational header={socialProfileHeader}>
+            <SocialEditableFieldContainer
```

Comment:
Each provider should only show if the site has a given provider configured

Reply:
how do we know if a site has a provider configured?